src/WebJobs.Script/Host/FunctionMetadataManager.cs

```diff
@@ -223,11 +223,12 @@ private void AddMetadataFromCustomProviders(IEnumerable<IFunctionProvider> funct
             }
 
             var functionMetadataListArray = Task.WhenAll(functionProviderTasks).GetAwaiter().GetResult();
+            var totalFunctionsCount = functionMetadataListArray.Sum(metadataArray => metadataArray.Length);
```

Comment:
recommend renaming "functionMetadataListArray" to "providerFunctionMetadataResults", since it's an array of arrays of FunctionMetadata by provider.

Reply:
Renamed as suggested (`providerFunctionMetadataResults` in the latest iteration pushed).